src/libsyntax/feature_gate.rs

```diff
@@ -441,10 +441,7 @@ declare_features! (
 
     // Allows macro invocations on modules expressions and statements and
     // procedural macros to expand to non-items.
-    (active, proc_macro_mod, "1.27.0", Some(38356), None),
-    (active, proc_macro_expr, "1.27.0", Some(38356), None),
-    (active, proc_macro_non_items, "1.27.0", Some(38356), None),
-    (active, proc_macro_gen, "1.27.0", Some(38356), None),
+    (active, proc_macro_hygiene, "1.27.0", Some(38356), None),
```

Comment:
Could you move the features into the "removed" list instead of just removing, and also add "subsumed by `proc_macro_hygiene`" as the removal reason?

Reply:
@bors r-